packages/editor/src/plugins/fill-in-the-blanks-exercise/components/blank-controls.tsx

```diff
@@ -116,6 +126,27 @@ export function BlankControls(props: BlankControlsProps) {
         className="w-[460px] rounded bg-white p-side text-start not-italic shadow-menu"
         style={{ width: `${wrapperWidth}px` }}
       >
+        {isBlankAnswerAlphabetical ? null : (
+          <div className="mb-6 flex">
+            <input
+              className="w-0.25 opacity-0"
+              id={'acceptMathEquivalentCheckbox_' + blankId}
```

Comment:
if you can wrap the input inside the label you don't need the `id` and `labelFor`.